plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/bazelrc/completion/BazelrcConfigDeclarationsProcessor.kt

```diff
@@ -0,0 +1,45 @@
+package org.jetbrains.bazel.languages.bazelrc.completion
+
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.util.Processor
+import org.jetbrains.bazel.languages.bazelrc.psi.BazelrcElement
+import org.jetbrains.bazel.languages.bazelrc.psi.BazelrcFile
+import org.jetbrains.bazel.languages.bazelrc.psi.BazelrcLine
+import org.jetbrains.kotlin.utils.addToStdlib.ifTrue
+
+class BazelrcConfigDeclarationsProcessor(private val inputElement: BazelrcLine) : Processor<BazelrcElement> {
+  var seenFiles = mutableSetOf<VirtualFile>()
+  var results = mutableMapOf<String, BazelrcFile>()
+
+  override fun process(currentElement: BazelrcElement): Boolean {
+    if (currentElement == inputElement) return true
+    processElement(currentElement)
+    return true
+  }
+
+  fun processFile(file: BazelrcFile?) {
+    file?.virtualFile?.apply {
+      seenFiles.add(this).ifTrue {
```

Comment:
`ifTrue` seems to come from some weird package, could you remove it please?

Reply:
The package started with org.jetbrains :) that automatically doesn't make it weird :) Fixed.